config/locales/es.yml

```diff
@@ -1522,7 +1522,7 @@ es:
 
               Si aún no has pagado mediante débito directo al presentar tu declaración, por favor haz tu pago lo antes posible para evitar multas y costos adicionales en %{state_pay_taxes_link}.
 
-              Descarga tu declaración en <a href=\"%{return_status_link}\" target=\"_blank\" rel=\"noopener nofollow\">%{return_status_link}</a>.
```

Comment:
b/c the `"` was being escaped, `"%{return_status_link}"` was seen as part of a url string and the default behavior for a random string without hostname is to concatenate it to the current host url + append?